### DISCUSSION TITLE
I can´t change theme settings.

### DISCUSSION THREAD

**Discussion by arthuresc** [psudo-dev/nebula-oni-theme#6] [Q&A]
![error nebula oni](https://github.com/psudo-dev/nebula-oni-theme/assets/48338321/d61ec09e-75b8-47b8-8607-900b24e3cb8c)

**Comment by psudo-dev** [comment]
I noticed that too, I might have to roll back the update. Did you use the previous version, was it working as it's supposed to? I haven't updated in a while, some things might have changed and it broke the menu settings when I made some minor corrections. Thanks

**Comment by psudo-dev** [comment]
I think I've fixed the issue, thanks!

**Reply by arthuresc** [reply]
Sorry, i just saw it right now haha, but now its working, really thanks. Mas obrigado mesmo mano, não sei se é real, mas tbm duvido que um gringo colocaria que é de Sp só pq é muito fã de SP haha

 Mano só passando pra dizer tbm que esse é o NebulaOni é o melhor tema mano, na moral baita trampo.

**Reply by psudo-dev** [reply]
Opa, valeu cara! Por motivos de força maior eu tô há um tempo sem mexer em nada, já tinha esquecido como mexe em tudo, fui tentar fazer funcionar na versão browser e acabei criando um bug - e eu nem sabia mais como que eu poderia corrigir, mas mesmo que na gambiarra, eu consegui.

Esse foi meu primeiro projeto depois que fiz um curso de JS, mas tive que aprender tudo na marra e demorei uns meses pra finalizar. No começo ia ser um tema de Atom One Dark modificado e sem querer virou algo completamente diferente. Aí botei na cabeça que queria oferecer algumas opções que normalmente os temas não tem e virou o que virou.

Eu ainda preciso dar um jeito de fazer tudo funcionar na versão browser e oferecer uma Night Version com texto mais escuro pra quem prefere baixo contraste. Tô voltando pros estudos agora então ainda vai demorar um tempo porque já esqueci quase tudo que aprendi... hahahaha...
